bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_ttree.py

```diff
@@ -15,6 +15,16 @@ def _TTree__iter__(self):
     if bytes_read == -1:
         raise RuntimeError("TTree I/O error")
 
+def _SetBranchAddress(self, *args):
+    # Modify the behaviour if args is (const char*, void*)
+    res = SetBranchAddressPyz(self, *args)
+
+    if res is None:
```

Comment:
As discussed in person, it seems that it is actually recommended the way it is now:
https://www.python.org/dev/peps/pep-0008/

`if not res` would call the `__bool__` operator on `res`, so it might be actually clearer to just compare to `None`.